root/etc/e-smith/templates/etc/httpd/conf.d/zzz_SOGo.conf/10base

```diff
@@ -1,88 +1,92 @@
 {
-    my $sogo_host = ($sogod{'VirtualHost'} || 'Default');
-    my $sogo_cert = ($sogod{'Certificate'} || '');
-    my $lib = 'lib64';
-	if (! -e '/usr/lib64/GNUstep/SOGo/') {
-	    $lib = 'lib';
-	}
-	my $end_config = ' ';
-    if (! (($sogod{'status'} || '') eq 'enabled')) {
-	$OUT .= "SOGo is disabaled\n";
-    }
-    else{
-        if ($sogo_host ne 'Default'){
-            my $chain_file = ' ';
-            $end_config = '</VirtualHost>';
-            if ($sogo_cert =~ /\./) {
-                $sogo_cert = (split(/\./, $sogo_cert))[0];
-             }
-            if ( -e "/etc/pki/tls/certs/$sogo_cert.crt") {
-                if ( -e "/etc/pki/tls/certs/$sogo_cert-chain.crt"){
-                   $chain_file = "SSLCertificateChainFile \"/etc/pki/tls/certs/$sogo_cert-chain.crt\"\n";
-                }   
-             }elsif ( -e '/etc/pki/tls/certs/Sogo.crt') {
-                $sogo_cert = 'Sogo';
-                if ( -e '/etc/pki/tls/certs/Sogo-chain.crt'){
-                   $chain_file = "SSLCertificateChainFile \"/etc/pki/tls/certs/Sogo-chain.crt\"\n";
-                }
-            } else {
-                $sogo_cert = 'localhost';
+my $status = $sogod{'status'} || 'disabled';
+return "#SOGo is disabled\n" unless ($status  eq 'enabled');
+
+my @sogo_hosts = split (',', ($sogod{'VirtualHost'} || 'Default'));
```

Comment:
Are we (re)introducing multiple vhost?
If so, 
How to handle the certificates for those hosts? 
Actually the reason it was changed to one single option, 
although that was before let's encrypt was fully developed.

Reply:
What is the use case of multiple vhost? Is it really needed? 
Can we make it simpler?

Reply:
Maybe I'm missing soething but... Isn't the ActiveSync endpoint defined as a "global" alias in Apache? so it can be accessible with any host name!

Reply:
>the documentation stated that multiple vhost is possible, I had two solutions, modify the documentation to say only one vhost is usable, this is the simplest way.

I took this functionally out a few years ago (for reason above), do not have the impression it has much demand.

Doc was/is changed about a month ago and made it to `latest',   
however it does not seem to land in v7

http://docs.nethserver.org/en/latest/sogo.html#access-sogo-on-an-exclusive-hostname

another minor argument to keep the single host is preventing  users wanting to setup a  fully multi tenant configuration. (eg `mail.ltdA.com` and `mail.ltdB.com`) which is not really possible with our accounts provider(s).

>mail.subdomain.com, but for a web application it is probably a non sense, it will probably sogo.subdomain.com and you will fail to configure your mobile activesync settings. It is a real case I got it this weekend after I did a reset to my cell

This makes sense and could be even needed for autodiscover. 
For the latter we could add 'autodiscover.yourdomain.com' in the future.

_in conclusion:_
not a deal breaker for me. IMO we are making it complicated and introduce noise on expectations on configuration you can achieve.

Reply:
> Maybe I'm missing soething but... Isn't the ActiveSync endpoint defined as a "global" alias in Apache? so it can be accessible with any host name!

IIRC at the moment is not, just a proxy pass, could we a good idea though